docs/source/developers/python.rst

```diff
@@ -270,6 +270,11 @@ Now build and install the Arrow C++ libraries:
          -DARROW_FLIGHT=ON \
          -DARROW_GANDIVA=ON \
          -DARROW_ORC=ON \
+         -DARROW_WITH_ZLIB=ON \
```

Comment:
BZ2 as well?

Reply:
Adding -- I had only listed the compressors used for Parquet